zetaclient/chains/sui/client/client_live_test.go

```diff
@@ -183,6 +184,19 @@ func TestClientLive(t *testing.T) {
 		require.NotNil(t, res)
 		require.Equal(t, TxStatusFailure, res.Effects.Status.Status)
 	})
+
+	t.Run("GetObjectParsedData", func(t *testing.T) {
+		// ARRANGE
+		ts := newTestSuite(t, RPCTestnet)
+
+		// ACT
+		objectID := "0x6fc08f682551e52c2cc34362a20f744ba6a3d8d17f6583fa2f774887c4079700"
```

Comment:
This is the gateway object? Can we mention it in comment?

Reply:
https://github.com/zeta-chain/node/pull/3850/commits/4bdf29bd602d81ff24b5577c12d9ada419f6975b